tests/features/tokens/login.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Login to Repository Service for TUF (RSTUF) and receive a access token
+    As an admin,
```

Comment:
The idea is to mention that the admin will run this feature.
We don't mention the login because it is the test scope.

> As an admin,
> Admin has deployed and bootstrapped RSTUF,
> Login using RSTUF API /api/v1/token POST method,
> Login using RSTUF API /api/v1/token POST method with an invalid username/password,
> Login using RSTUF API /api/v1/token POST method get Forbidden for invalid scopes,
> Login using RSTUF CLI,
> Login using RSTUF CLI get Unauthorized.